.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+.github/CODEOWNERS @SonarSource/languages-team-dotnet
```

Comment:
If I remember correctly this tool is used by the sonar lint team too. @duncanp-sonar would you like to be added as an owner too?